airbyte-integrations/connectors/source-github/source_github/streams.py

```diff
@@ -57,6 +58,9 @@ def __init__(self, api_url: str = "https://api.github.com", access_token_type: s
         self.api_url = api_url
         self.state = {}
 
+        if self.cursor_field or len(self.cursor_field) == 0:
```

Comment:
I'm a little confused by this check - we use the substream RFR cursor if there is a cursor_field or if its length is 0?

Reply:
yep, sorry i should've commented, but it's supposed to be `not self.cursor_field`, I reversed it to test something on regression tests, but i'll change it back for final testing when i move it out of draft

Reply:
Why not use the `self.supports_incremental` property to be checked?

[CDK Code](https://github.com/airbytehq/airbyte/blob/baz/source/shopify/fix-faild-bulk-and-deleted-events/airbyte-cdk/python/airbyte_cdk/sources/streams/core.py#L305)

It could be simply as: 
```python
if not self.supports_incremental:
    ...
```

Reply:
yep fair point, will change thanks for the suggestion!